Automattic/jetpack
# Jetpack: improve QR Post code site/Jetpack logo

This PR tweaks the QR post code logo either for the site logo, or for the Jetpack logo when the site doesn't have it.

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: improve QR Post code site/Jetpack logo

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Edit a post via Block editor
* Take a look at the QR Post code 
* Confirm the following visual changes

Without site logo

before | after
------|------
<img width="418" alt="image" src="https://user-images.githubusercontent.com/77539/158612832-c00691bb-e397-4457-b040-a0e8eb671a7d.png"> | <img width="411" alt="image" src="https://user-images.githubusercontent.com/77539/158612486-63fd7102-63c4-4f98-9783-06ad1b37851e.png">

With site logo

before | after
------|------
<img width="413" alt="image" src="https://user-images.githubusercontent.com/77539/158612743-0bbc3643-8be5-48c7-a06d-fbff948a3d63.png"> | <img width="417" alt="image" src="https://user-images.githubusercontent.com/77539/158612598-e501ab85-8542-4886-9643-435144680ef4.png">
